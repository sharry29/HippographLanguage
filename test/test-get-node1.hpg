int main() {
	graph<int:string> g = [1:"foo"; 2:"bar"];
	node<int:string> n1 = g.get_node(1);
	node<int:string> n2 = g.get_node(2);
	node<int:string> n3 = g.get_node(3);

	n1.print();
	print("");
	n2.print();
	print("");
	n3.print();
}