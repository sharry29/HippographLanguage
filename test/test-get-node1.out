1:"foo"
2:"bar"
0:null